ecoscope/io/eetools.py

```diff
@@ -4,13 +4,16 @@
 import itertools
 import json
 import logging
+from collections.abc import Callable
+from typing import cast
```

Comment:
```suggestion
from typing import Callable, cast
```
Small point but maybe this is simpler